### ISSUE TITLE
Document standards/rationale for information excluded from citizens content

### ISSUE THREAD

**Issue by AmyKort** [18F/fec-cms#1132]
The FEC provides a number of publications and pages target toward the general public. In creating a single set of pages for this content, we'll have to retire or find a new home for some of this information--whether because it is redundant or because it is not a top priority for users of these pages. For future reference and to ensure continuity, we need to document standards for the content to be included in this new section. Related to https://github.com/18F/fec-cms/issues/1122 and https://github.com/18F/fec-cms/issues/1124

- [x] Create a plan for treating cyclical content.
- [x] Look at past website usage to determine what's popular when.

**Comment by AmyKort** [comment]
We'll apply this to the citizens guide related content.

**Comment by AmyKort** [comment]
Since we've determined that our current universe of citizens content may actually need to live in separate areas of the website, we're focusing on just the guidance provided in the citizens guide, former quick answers, etc. We've competed the work of prioritizing this content and documented our decision making process. We'll take separate steps to account for other content, such as guidance on how to research our public data records and information for how/when to contact other agencies for help and information.
